src/App.tsx

```diff
@@ -1,14 +1,97 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
+// eslint-disable-next-line import/no-cycle
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos, getUser } from './api';
+
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+export enum FilterValues {
+  All = 'All',
+  Active = 'active',
+  Completed = 'completed',
+}
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [loadedTodos, setLoadedTodos] = useState(false);
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [loadedModal, setLoadedModal] = useState(false);
```

Comment:
do you really need this state in the App? can we store it in the TodoModal?

Reply:
I don't know how implement it, because I set this state in the App